web/.eslintignore

```diff
@@ -3,4 +3,3 @@ web/src/api/generated
 
 build
 coverage
-web/src/components/TodoApp.tsx
```

Comment:
Good that you removed this. Next time, maybe we should do it in a separate commit to keep order in the git history. I don't see why this change belongs under the title `fix: broken auth under local environment`